crates/walrus-service/src/node/errors.rs

```diff
@@ -161,6 +161,11 @@ pub enum ComputeStorageConfirmationError {
     #[rest_api_error(reason = "NOT_REGISTERED", status = ApiStatusCode::FailedPrecondition)]
     NotCurrentlyRegistered,
 
+    /// The pending upload cache is full and cannot accept additional metadata.
+    #[error("pending upload cache is saturated; retry once the node catches up")]
+    #[rest_api_error(reason = "CACHE_SATURATED", status = ApiStatusCode::FailedPrecondition)]
+    CacheSaturated,
+
```

Comment:
Question: Why is this relevant for the storage confirmation?

Reply:
it is not, removed.